clusterproperty/config/samples/about_v1alpha1_clusterproperty.yaml

```diff
@@ -0,0 +1,9 @@
+# An example object of `id.k8s.io ClusterProperty` 
+# using a kube-system ns uuid as the id value (recommended above):
```

Comment:
tiny nit:

```suggestion
# using a kube-system ns uuid as the id value:
```

Reply:
@lauralorenz : Thank you for the suggestion. Updated the file per suggestion.